inc/saiport.h

```diff
@@ -47,6 +47,19 @@ typedef enum _sai_port_type_t
 } sai_port_type_t;
 
 /**
+* @brief Attribute data for #SAI_PORT_ATTR_BIND_MODE
+*/
+typedef enum _sai_port_bind_mode_t
+{
+    /** Port */
+    SAI_PORT_BIND_MODE_PORT,
+
+    /** Sub port */
+    SAI_PORT_BIND_MODE_SUB_PORT,
+
+} sai_port_bind_mode_t;
```

Comment:
currently we don't have admin state for all  logical interfaces E.g router interface 
I agree that we need to consider adding admin state to all logical interfaces including router interface and bridge interface . 
but i don't  think it is mandatory for 1.0 we can the Admin  state attribute leter do you agree ?